### PR TITLE
bumps the tx execution to info

### PR DESCRIPTION
**Pull request by otherview**
### Why is this change needed?

- Log correctly executed txs

### What changes were made as part of this PR:

- tx log

### :rotating_light: Definition of Done :rotating_light:
- [ ] Deployed into dev-testnet 
- [ ] Passes tests on dev-testnet
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

File: go/common/logging.go [hunk 1]
```diff
@@ -49,6 +49,12 @@ func LogTXExecution(txHash common.Hash, msg string, args ...interface{}) {
 	log.Info(txExecPattern, txHash.Hex(), formattedMsg)
 }
 
+// DebugTXExecution - logs at DEBUG level with the tx hash prepended
+func DebugTXExecution(txHash common.Hash, msg string, args ...interface{}) {
```
Comment (tudor-malene) [inline go/common/logging.go:53]: what's up with this? 
If not used, pls remove

**Review by tudor-malene — COMMENTED** [review]
one comment

File: go/common/logging.go (hunk 1 above)
Comment (otherview) [reply go/common/logging.go:53]: Yep 👍  I decided last minute just bump this to log level.

**Review by tudor-malene — APPROVED** [review]
lgtm

**Pull request merged**